examples/kuesa/car-scene/qml/main.qml

```diff
@@ -665,4 +668,51 @@ Item {
         loops: Animation.Infinite
         running: idleAnimationRunning
     }
+
+    // Title Panel & Animation
```

Comment:
I'd move this in a separate file as well to keep main.qml from exploding. I'd even be tempted to move the whole side panel into a dedicated component so that main.qml is really about instantiating the 3D view and the various UI panels

Reply:
+1

Reply:
done, moved menu out of main, and animation also in separate file